packages/react-ui/components/Autocomplete/__stories__/Autocomplete.stories.tsx

```diff
@@ -114,7 +232,53 @@ WithFixedMenuSize.story = {
     },
   },
 };
+//Появление скролла при большом кол-ве элементов + его расширение при наведении
+export const AutocompleteWithScroll: CSFStory<JSX.Element> = () => {
+  const items = [];
+  for (let i = 0; i < 20; i++) {
+    items.push(`Abba ${i}`);
+  }
+  return (
+    <div style={{ padding: '4px 200px 200px 4px' }}>
+      <UncontrolledAutocomplete source={items} />
+    </div>
+  );
+};
+AutocompleteWithScroll.story = {
+  name: 'with scroll',
+  parameters: {
+    creevey: {
+      tests: {
+        async scrollBarShouldBeVisible() {
+          const element = await this.browser.findElement({ css: '#test-element' });
+          const autocomplete = await this.browser.findElement({ css: '[data-comp-name~=Autocomplete]' });
+
+          await this.browser
+            .actions({ bridge: true })
+            .click(autocomplete)
+            .perform();
+
+          await this.browser
+            .actions({ bridge: true })
+            .sendKeys('Abba')
+            .perform();
```

Comment:
Подобные конструкции можно сократить. 
```suggestion
          await this.browser
            .actions({ bridge: true })
            .click(autocomplete)
            .sendKeys('Abba')
            .perform();
```

---

Метод `.actions({ bridge: true })` возвращает экземпляр класса `Actions`, и почти каждый метод этого класса тоже возвращает этот экземпляр.
Поэтому несколько действия подряд можно оформить цепочкой:
```javascript
.actions({ bridge: true })
.click()
.sendKeys()
.press()
.move()
...
.perform();
```